packages/teams-js/test/public/media.spec.ts

```diff
@@ -838,7 +713,146 @@ describe('media', () => {
           },
         } as DOMMessageEvent);
 
-        return expect(promise).rejects.toEqual({ errorCode: ErrorCode.SIZE_EXCEEDED });
+        await expect(promise).rejects.toMatchObject({ errorCode: ErrorCode.SIZE_EXCEEDED });
+      });
+    });
+  });
+
+  describe('videoController', () => {
+    describe('v1', () => {
+      it('videoController notifyEventToHost is handled successfully', async () => {
+        expect.assertions(7); // initializeWithContext has 3 assertions + 4 in this test = 7
```

Comment:
I love these comments. It makes it a lot easier to understand where the hidden assertions are. (Not a huge fan of hidden assertions but not something you introduced here so no need to go fix it as part of this change)

Reply:
i agree, this confused me a LOT when i was writing the tests, I'm not sure why we have assertions in a mock, but that's something we can discuss outside of this PR